Simplenote/SPAppDelegate.m

```diff
@@ -230,8 +230,6 @@ - (void)setupCrashlytics
     NSString *email = self.simperium.user.email;
     NSString *key = [SPCredentials simplenoteCrashlyticsKey];
 
-    [Fabric with:@[CrashlyticsKit]];
```

Comment:
I think we still need this no? Looking at the install guide here (albiet with possibly a small modification): https://fabric.io/kits/ios/crashlytics/install

`[Fabric with:@[[Crashlytics class]]];`

Reply:
I think you're right! I thought I read somewhere that it wasn't needed. But I added it back and I'm still seeing the crashes being sent:

```[Crashlytics:Crash:Reports] Submitting async /var/mobile/Containers/Data/Application/D48F905D-C8A2-45A1-846E-92B1517C6D1F/Library/Caches/com.crashlytics.data/com.codality.NotationalFlow/v3/prepared/BE27225B-B740-4745-85D5-A4FF00B3C853.multipartmime```

Reply:
Ok thanks. I'd prefer it stay in just in case since the documentation says to do it.